internal/master/service.go

```diff
@@ -260,6 +267,28 @@ func (ss *standaloneService) Restore(ctx context.Context, restoreReq *serverpb.R
 	return newEmptyStatus(), nil
 }
 
+func (ss *standaloneService) PrefixMultiGet(ctx context.Context, prefixMultiGetRequest *serverpb.PrefixMultiGetRequest) (*serverpb.MultiGetResponse, error) {
+	ss.rwl.RLock()
+	defer ss.rwl.RUnlock()
+
+	// Convert prefix multi get request to prefix iterator and append the results to construct the response
+	iterReq := &serverpb.IterateRequest{KeyPrefix: prefixMultiGetRequest.KeyPrefix}
+	iteration := storage.NewIteration(ss.store, iterReq)
+
+	var results []*serverpb.KVPair
+
+	err := iteration.ForEach(func(k, v []byte) error {
+		results = append(results, &serverpb.KVPair{Key: k, Value: v})
+		return nil
+	})
+	if err != nil {
+		// Better to return error rather than partial results as incomplete results could cause client misbehaviour
+		ss.lg.Error("Unable to PrefixMultiGet", zap.Error(err))
+		return &serverpb.MultiGetResponse{Status: newErrorStatus(err)}, err
+	}
+	return &serverpb.MultiGetResponse{KeyValues: results, Status: newEmptyStatus()}, nil
+}
```

Comment:
since `PrefixMultiGet` does nothing but just calls `IterateRequest`, let remove this, and let the caller exactly do this

Reply:
Fixed in #81